src/main/java/com/yahoo/memory/WritableBufferImpl.java

```diff
@@ -51,66 +51,82 @@
   final static WritableBufferImpl ZERO_SIZE_BUFFER;
 
   static {
-    ZERO_SIZE_BUFFER = new WritableBufferImpl(new ResourceState(new byte[0], Prim.BYTE, 0));
+    ZERO_SIZE_BUFFER = new WritableBufferImpl(new ResourceState(new byte[0], Prim.BYTE, 0), true);
     ZERO_SIZE_BUFFER.originMemory = WritableMemoryImpl.ZERO_SIZE_MEMORY;
   }
 
-  WritableBufferImpl(final ResourceState state) {
-    super(state);
+  WritableBufferImpl(final ResourceState state, final boolean localReadOnly) {
+    super(state, localReadOnly);
   }
 
+  //  static WritableBufferImpl newInstance(final ResourceState state, final boolean localReadOnly) {
```

Comment:
garbage

Reply:
At first I thought I needed this, but I don't.  Will remove.